MOBILE_CLIENT/COMMON/_HIGH/OAUTH_ACCOUNT_TAKEOVER/description.md

```diff
@@ -0,0 +1,149 @@
+The vulnerability arises from the application's use of a custom scheme in the `redirect_uri` parameter during OAuth authentication. 
```

Comment:
If the finding is only for OAuth account takeover using custom scheme, then the title of the entry should be changed.

I would suggest keeping this generic and list the ways that can happen on Mobile Apps.

Custom URL scheme.
Localhost.
Open redirect on the receiving endpoint.

and explain each.